src/pages/ScholarshipForm.jsx

```diff
@@ -1,21 +1,64 @@
 import React, { useState } from 'react';
 import { Container, Button, TextField } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
 import ScholarshipAmountField from '../components/ScholarshipAmountField';
 import DatePicker from '../components/DatePicker';
+import { invalidAmountFields } from '../validation/amountValidation';
+
+const useStyles = makeStyles({
+  textFieldStyle: {
+    minWidth: 400,
+  },
+});
+
+// regular expression used for valid website urls
+const websiteReg = /^(http(s)?:\/\/(www\.)?)?[\w-]+(\.[\w-]+)*\.[a-z]{2,}((\/|#|\?)(\S)*)?$/;
 
 function ScholarshipForm() {
+  const classes = useStyles();
   const [formFieldStates, setFormFieldStates] = useState({
```

Comment:
(inspired by reading Formik docs) you could shorten these names too:
```suggestion
  const [values, setValues] = useState({
```

And could also, if you like, replace lines like these:
```jsx
onChange={updateFn('name')}
```
with
```jsx
onChange={(e) => setValues({values..., name: e.target.value})}
```
If that makes things easier to read